src/luascript.cpp

```diff
@@ -12608,6 +13360,383 @@ int LuaScriptInterface::luaPartySetSharedExperience(lua_State* L)
 	return 1;
 }
 
+// Loot
+
+int LuaScriptInterface::luaCreateLoot(lua_State* L)
+{
+	// Loot() will create a new loot item
+	Loot* loot = new Loot();
```

Comment:
on the newest commit I tested it, I'm reloading lua monsters aswell there, it either re adresses stuff or it gets cleaned by GC cause as I said, there is no memory increase or w/e tested this now for quite a while and it seemed fine.